assignments/arrays.js

```diff
@@ -63,34 +63,66 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+let keys = Object.keys(inventory);
+let len = keys.length;
+// for (let i=0; i<len; i++) {
+//   console.log(inventory[i]);
+// }
+
+console.log('Car 33 is a ' + inventory[33].car_year.toString()
```

Comment:
```
.toString()
```
This is not necessary. If we look at the data types, most of them are strings already and the console will print them without making them into strings. Very minor, we just don't need to be this explicit.